@navikt/ds-css/modal/index.scss

```diff
@@ -47,7 +51,9 @@
   position: absolute;
   top: 1rem;
   right: 1rem;
+  display: flex;
```

Comment:
Hvorfor trengte vi plutselig denne?

Reply:
Var slik for å sentrere modalvinduet vertikalt. Erstattet det med 
```css
top: 50%;
transform: translateY(-50%);
```
Men vil gjøre det samme så ikke for mye verdi i det